Madness.xcodeproj/project.pbxproj

```diff
@@ -7,16 +7,19 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		849DC2381C0F21D0004C1A1E /* Combinator.swift in Sources */ = {isa = PBXBuildFile; fileRef = 849DC2371C0F21D0004C1A1E /* Combinator.swift */; };
+		849DC2391C0F21D0004C1A1E /* Combinator.swift in Sources */ = {isa = PBXBuildFile; fileRef = 849DC2371C0F21D0004C1A1E /* Combinator.swift */; };
+		849DC23A1C0F21E1004C1A1E /* String.swift in Sources */ = {isa = PBXBuildFile; fileRef = D1E153551BD5B78E00627E39 /* String.swift */; };
```

Comment:
`String.swift` was disabled for which targets. So, I added `String.swift` to `Compile Sources/Build Phases`.